examples/java/spring-boot/src/test/java/uk/gov/api/springboot/application/services/EmptyFetcherServiceTest.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.api.springboot.application.services;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.api.springboot.domain.model.Registry;
+
+class EmptyFetcherServiceTest {
+
+  private final EmptyFetcherService fetcherService = new EmptyFetcherService();
+
+  @Test
+  void fetchReturnsAnEmptyList() {
+    var registryEntry = new Registry.Entry("123", "www.example.com");
+
+    assertThat(fetcherService.fetch(registryEntry)).isEmpty();
```

Comment:
This makes it a little clearer the method is doing nothing:

```suggestion
    assertThat(fetcherService.fetch(null)).isEmpty();
```